plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -769,4 +780,18 @@ private void updateAllDependents(String parent, Plugin dependent, Map<String,Ver
         }
         return diff;
     }
+
+    private boolean checkJenkinsTestHarness(File buildLogFile, MavenPom pom) throws IOException {
+        try (FileInputStream is = new FileInputStream(buildLogFile)) {
+            String log = IOUtils.toString(is);
+
+            if (log.contains("Failure to find org.jenkins-ci.main:jenkins-test-harness")) {
+                pom.replaceDependencyVersion("org.jenkins-ci.main", "jenkins-test-harness", "RELEASE");
```

Comment:
To clarify in case someone could have the same doubt: that would not fix in case the plugin explicitly had the dependency without using `jenkins-test-harness.version` property.

```
<dependency>
    <artifactId>jenkins-test-harness</artifactId>
    <version>${jenkins.version}</version>
</dependency>
```